website/docs/cli/cloud/migrating.mdx

```diff
@@ -82,3 +82,4 @@ terraform {
    }
  }
 ```
+~> **Warning:** Because the `cloud` block does not support the `prefix` argument, once you migrate, you must refer to workspaces by their full name when using the Terraform CLI. For example, rather than `terraform workspace select prod`, you must run the command `terraform workspace select my-app-prod`.
```

Comment:
Hello! 👋 Sorry for the slow response here, I'm going through our backlog of PRs and I'd like to check this out in a Vercel preview (with a line for styling between the code block and the warning you are suggesting). In the interest of expediting this, I'm going to add this small commit before approving: 
```suggestion

~> **Warning**: Because the `cloud` block does not support the `prefix` argument, once you migrate, you must refer to workspaces by their full name when using the Terraform CLI. For example, rather than `terraform workspace select prod`, you must run the command `terraform workspace select my-app-prod`.
```